webapp/src/main/java/com/box/l10n/mojito/entity/RepositoryAIPrompt.java

```diff
@@ -0,0 +1,49 @@
+package com.box.l10n.mojito.entity;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Index;
+import jakarta.persistence.Table;
+
+@Entity
+@Table(
+    name = "repository_ai_prompt",
+    indexes = {
+      @Index(name = "I__REPOSITORY_AI_PROMPT__REPOSITORY_ID", columnList = "repository_id"),
+      @Index(name = "I__REPOSITORY_AI_PROMPT__PROMPT_TYPE_ID", columnList = "prompt_type_id")
+    })
+public class RepositoryAIPrompt extends BaseEntity {
+
+  @Column(name = "repository_id")
+  private Long repositoryId;
+
+  @Column(name = "ai_prompt_id")
+  private Long aiPromptId;
+
+  @Column(name = "prompt_type_id")
+  private Long promptTypeId;
```

Comment:
To clarify a data modelling question: the expectation implemented here is that a repo can have multiple prompts for the same prompt_type_id, is that correct? (If not, we should have a unique index to enforce uniqueness)

Reply:
Yes that's correct, a repository could have multiple string checker prompts, each of which is used to check a string for issue.

When it comes to translation prompts it doesn't make sense to have multiple active 'translate' prompts for a repository so that'll need to be restricted at the application level.